test/application_system_test_case.rb

```diff
@@ -1,5 +0,0 @@
-require "test_helper"
-
-class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
```

Comment:
hmm. I am not sure if we wanna remove this. I feel if we wanna add more tests, it would be really useful if the test can be integration tests. So that writing one test can catch many cases in the whole flow.